src/autoupdater.ts

```diff
@@ -265,6 +265,21 @@ export class AutoUpdater {
       }
     }
 
+    const readyStateFilter = this.config.pullRequestReadyState();
+    if (readyStateFilter !== 'all') {
+      ghCore.info('Checking PR ready state');
+
+      if (readyStateFilter === 'draft' && !pull.draft) {
+        ghCore.info('Pull request is not draft, skipping update.');
```

Comment:
Just a minor recommendation to make it slightly more verbose:
```suggestion
        ghCore.info('PR_READY_STATE=draft and pull request is not draft, skipping update.');
```